cyberdrop_dl/utils/sorting.py

```diff
@@ -88,55 +91,63 @@ async def sort(self) -> None:
             await log_with_color("Download Directory does not exist", "red", 40)
             return
 
+        download_folders=await self.get_download_folder()
+        async with self.manager.live_manager.get_sort_live(stop=True):
+            all_scan_folders=list(filter(lambda x:x.is_dir(),self.download_dir.iterdir()))
+            await self.manager.progress_manager.sort_progress.set_total(len(all_scan_folders))
+
+            for folder in all_scan_folders:
+                if self.sort_cdl_only and folder not in download_folders:
+                    pass
+                else:
+                    files = await self.find_files_in_dir(folder)
+                    for file in files:
+                        ext = file.suffix.lower()
+                        if '.part' in ext:
+                            continue
+
+                        if ext in FILE_FORMATS['Audio']:
+                            await self.sort_audio(file, folder.name)
+                        elif ext in FILE_FORMATS['Images']:
+                            await self.sort_image(file, folder.name)
+                        elif ext in FILE_FORMATS['Videos']:
+                            await self.sort_video(file, folder.name)
+                        else:
+                            await self.sort_other(file, folder.name)
+                        await purge_dir_tree(folder)
+                await self.manager.progress_manager.sort_progress.add_sorted_dir()
+
+        await asyncio.sleep(5)
+        await purge_dir_tree(self.download_dir)
+
+
+        await log_with_color(f"Organized: {self.audio_count} Audio Files", "green", 20)
+        await log_with_color(f"Organized: {self.image_count} Image Files", "green", 20)
+        await log_with_color(f"Organized: {self.video_count} Video Files", "green", 20)
+        await log_with_color(f"Organized: {self.other_count} Other Files", "green", 20)
+
+    async def get_download_folder(self):
+        """Gets the download folder"""
+        if not self.sort_cdl_only:
+            return []
         unique_download_paths = await self.db_manager.history_table.get_unique_download_paths()
-        download_folders = [Path(download_path[0]) for download_path in unique_download_paths if Path(download_path[0]).is_dir() and Path(download_path[0]) != self.download_dir]
+        download_folders = download_folders = [Path(download_path[0]) for download_path in unique_download_paths if Path(download_path[0]).is_dir() and Path(download_path[0]) != self.download_dir]
         existing_folders = []
-        
         for folder in download_folders:
             try:
                 relative_folder = folder.relative_to(self.download_dir)
                 base_folder = self.download_dir / relative_folder.parts[0]
+                print(relative_folder)
```

Comment:
Is this intended to be included in the PR?

Reply:
no that must have been from testing